contrib/jemalloc-cmake/CMakeLists.txt

```diff
@@ -164,14 +164,16 @@ target_compile_definitions(_jemalloc PRIVATE -DJEMALLOC_NO_PRIVATE_NAMESPACE)
 # Because our coverage callbacks call malloc, and recursive call of malloc could not work.
 target_compile_options(_jemalloc PRIVATE ${WITHOUT_COVERAGE_FLAGS_LIST})
 
-if (CMAKE_BUILD_TYPE_UC STREQUAL "DEBUG")
-    target_compile_definitions(_jemalloc PRIVATE
-        -DJEMALLOC_DEBUG=1
-        # Usage examples:
-        # - MALLOC_CONF=log:.
-        # - MALLOC_CONF='log:core.malloc.exit|core.sallocx.entry|core.sdallocx.entry'
-        -DJEMALLOC_LOG=1)
-endif ()
+# We don't enable Jemalloc DEBUG and log anymore since those have a big impact in performance, specially noticeable
+# when starting the process (really common in stateless tests with the client)
+#if (CMAKE_BUILD_TYPE_UC STREQUAL "DEBUG")
+#    target_compile_definitions(_jemalloc PRIVATE
+#        -DJEMALLOC_DEBUG=1
+#        # Usage examples:
+#        # - MALLOC_CONF=log:.
+#        # - MALLOC_CONF='log:core.malloc.exit|core.sallocx.entry|core.sdallocx.entry'
+#        -DJEMALLOC_LOG=1)
+#endif ()
```

Comment:
Should we delete it entirely?

Reply:
Sure. I initially left it thinking it might show some obscure ways of configuring jemalloc, but it's not and it's not like anybody uses it.